Module1.FileSystemVisitor/Tests/Tests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Project;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Tests
+{
+    [TestClass]
+    public class Tests
+    {
+        public string CurrentDirectory
```

Comment:
what about such getter?
```suggestion
        public string CurrentDirectory => Directory.GetParent(Directory.GetCurrentDirectory()).FullName;
```

Reply:
done